tests/src/Functional/OpenVocabularyAssociationAdminTest.php

```diff
@@ -21,6 +24,7 @@ class OpenVocabularyAssociationAdminTest extends BrowserTestBase {
    * {@inheritdoc}
    */
   protected static $modules = [
+    'node',
```

Comment:
There's no node handler so nodes shouldn't be selectable. It's better to use the entity_test entities, so we can cover also scenarios for entity types without bundles.